src/zenml/cli/model.py

```diff
@@ -468,24 +468,30 @@ def _print_artifacts_links_generic(
     """
     model_version = Client().get_model_version(
         model_name_or_id=model_name_or_id,
-        model_version_name_or_number_or_id=model_version_name_or_number_or_id,
+        model_version_name_or_number_or_id=None
+        if model_version_name_or_number_or_id == "0"
+        else model_version_name_or_number_or_id,
+    )
+    type_ = (
+        "artifacts"
+        if only_artifacts
```

Comment:
Nitpick: "only_artifacts" doesn't really make sense semantically since the deployments and models are also artifacts